presto-main/src/main/java/com/facebook/presto/sql/relational/SqlToRowExpressionTranslator.java

```diff
@@ -653,12 +610,27 @@ protected RowExpression visitTryExpression(TryExpression node, Void context)
             throw new UnsupportedOperationException("Must desugar TryExpression before translate it into RowExpression");
         }
 
+        private RowExpression buildEquals(RowExpression lhs, RowExpression rhs)
+        {
+            return call(EQUAL.getOperator(),
```

Comment:
move `EQUAL.getOperator()` to its own line

Reply:
Done